test/dashboard-e2e/pages/MainPage.ts

```diff
@@ -7,11 +7,32 @@ export class MainPage{
     }
 
     async visitMainPage(){
-      await this.page.goto(`/apiEndpoint?apiEndpoint=${process.env.DASHBOARD_API_URL}`);
+      if (process.env.CLOUD_CONTEXT) {
+        const userToken = process.env.BEARER_TOKEN
+        const apiEndpoint = process.env.CLOUD_API_URL
```

Comment:
Why separate `CLOUD_API_URL` and `DASHBOARD_API_URL`? It will use only one URL in the same context - it won't call OSS cluster in Cloud, or Cloud in OSS

Reply:
There are separate  `DASHBOARD_API_URL` and `API_URL` because they can be set to different values. The first one is the one set as `apiEndpoint`, and the second one is used by API helpers.
For example,
```
DASHBOARD_API_URL="https://dev.testkube.io/results/v1"
API_URL="testkube-api-server.testkube.svc.cluster.local:8088/v1"
```

Reply:
Ahh, sorry - looks like it's merged already :)